bats/tests/snapshots/create-use-snapshot.bats

```diff
@@ -111,6 +111,11 @@ local_setup() {
     assert_output ''
 }
 
+@test 'factory-reset does delete an empty snapshots directory' {
+    rdctl factory-reset
+    assert_not_exists "$PATH_SNAPSHOTS_DIR"
+}
```

Comment:
At a later time I would like to see these tests moved to be with the factory-reset tests. But not right now.

Reply:
Except the factory-reset tests are all symmetrical: if X exists before factory-reset, it shouldn't after. That's not true for snapshots

Reply:
We can figure it out; it is just code. 😄 

Everything else done by factory-reset is tested in a single place; so it seems weird to have one particular thing tested elsewhere.

But we can discuss this later; so you get another chance to change my mind. 😄